galata/test/jupyterlab/inline-completer.test.ts

```diff
@@ -251,6 +251,47 @@ test.describe('Inline Completer', () => {
       await expect(ghostText).toBeHidden();
     });
 
+    test('Ghost text shows on middle of line when FIM is enabled', async ({
+      page
+    }) => {
+      //Enabling FIM
+      await page.evaluate(async () => {
+        await window.jupyterapp.commands.execute('settingeditor:open', {
+          query: 'Inline Completer'
+        });
+      });
+      await page
+        .locator(
+          '#jp-SettingsEditor-\\@jupyterlab\\/completer-extension\\:inline-completer_providers_\\@jupyterlab\\/inline-completer\\:history_autoFillInMiddle'
+        )
+        .click();
```

Comment:
This locator should use user-facing or accessibility-oriented features. Like select by having text matching "Fill in middle on typing". This means that if implementation of the settings editor changes (e.g. we update `rjsf`) then the test will still work.